src/test/java/org/folio/rest/api/HoldingsSourceTest.java

```diff
@@ -0,0 +1,116 @@
+package org.folio.rest.api;
+
+import static org.folio.rest.support.http.InterfaceUrls.holdingsSourceUrl;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.rest.support.IndividualResource;
+import org.folio.rest.support.Response;
+import org.folio.rest.support.ResponseHandler;
+import org.folio.rest.support.http.ResourceClient;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class HoldingsSourceTest extends TestBase {
```

Comment:
Please add this tests to `org.folio.rest.api.StorageTestSuite`, so that the test cases will be included into coverage.